docs/user_manual/source/control_status_registers.rst

```diff
@@ -1464,7 +1462,7 @@ Detailed:
 
   CSR Address: 0x747
 
-  Reset Value: 0x0000_0000
+  Reset Value: defined (based on ``PMP_MSECCFG_RV``)
```

Comment:
Wait a second... Isn't this 40s-specific?

Reply:
Is `.. only:: PMP` some "feature flag" that allows 40x/40s to share some documentation source but only be visible if the flag is enabled?

Reply:
Yes, correct. We try to use the same rst files as much as possible, but we have some conditional rendering. (It is not applied to the Xsecure CSRs, but I can introduce such a flag later as well to further reduce the diff between the two repos.